lib/hexpm/accounts/recovery_code.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Hexpm.Accounts.RecoveryCode do
+  use Hexpm.Schema
+
+  schema "user_recovery_codes" do
```

Comment:
I think this should be an `embeds_many` on whatever table we have to store 2FA info in the future. But since we don't have that table yet we can keep it like this and refactor later.

Reply:
That's fair. I didn't do too much as can be seen as I knew it would be in flux after the PR was opened.